index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" type="text/css" />
+    <link rel="stylesheet" href="./normalize.css" type="text/css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="links"><a href="#">PROJECTS</a></div>
+        <div class="links"><a href="#">ABOUT ME</a></div>
+        <div class="links"><a href="#">HIRE ME</a></div>
+      </div>
+    </header>
+
+    <div class="landing-panel">
+      <div class="container">
+        <h1>DANIEL GARDINER</h1>
+        <h2>JUNIOR DEVELOPER</h2>
```

Comment:
generally we advise not using qualifiers when referring to yourself. who says your a junior developer? by what standard? just developer, or software developer.